app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -665,7 +669,7 @@ class HomeFragment : Fragment() {
                                     isDisplayedWithBrowserToolbar = false
                                 )
                                     .setText(it.context.getString(R.string.onboarding_firefox_account_sync_is_on))
-                                    .setAnchorView(toolbarLayout)
+                                    // .setAnchorView(toolbarLayout)
```

Comment:
🤔

Reply:
Thanks! Updating to still use the anchor.